### DISCUSSION TITLE
Make Link to ReadTheDocs Typing Docs More Explicit

### DISCUSSION THREAD

**Discussion by adam-grant-hendry** [python/typing#1255] [Q&A] [answered]
When googling typing for Python, the `typing` module is the first thing that pops up and nothing from the `typeshed` ReadTheDocs:

https://typing.readthedocs.io/en/latest/#

The information here is INVALUABLE. I wish I had known it sooner! It wasn't until I happened upon it in the [`typeshed` github repo](https://github.com/python/typeshed/) that I saw it.

Could this PLEASE be made more explicit? (i.e. if possible, adding a link at the top of the `typing` module docs would be super!)

Besides the [`typing` module](https://docs.python.org/3/library/typing.html), [`mypy` docs](https://mypy.readthedocs.io/en/stable/getting_started.html), and [`pyright` github docs page](https://github.com/microsoft/pyright/blob/main/docs/type-concepts.md), I wasn't aware there was more information. This has a lot of details I was missing.

**Comment by hauntsaninja** [comment]
I'm glad you find it useful. Want to create a PR to CPython? Something like the the "See also" box on https://docs.python.org/3/library/re.html would work nicely

**Reply by adam-grant-hendry** [reply]
Thank you @hauntsaninja ! I would love to, but currently do not have the bandwidth for this and am not sure when I will. I bet you're also in the same boat 😆 .

Should I find some extra time, I would be very glad to submit a PR, but for now I cannot promise my time on anything.

**Comment by adam-grant-hendry** [comment, marked as answer]
@hauntsaninja I would say the answer should be these:

- [Proposal: Convert the typing-module PEPs into a series of HOWTOs](https://github.com/python/cpython/issues/91533)
- [Document Structure and Wishlist](https://github.com/python/typing/issues/845)

Efforts are being made there to help this situation. I think it will be better to contribute discussions and efforts to these rather than attempt making a separate PR to CPython.

I highly recommend new users vote (thumbs-up/thumbs-down/etc.) if they feel a comprehensive series of typing HOW-TO's will benefit them. Upvotes improve the chances of it succeeding!

**Reply by AlexWaygood** [reply]
> rather than attempt making a separate PR to CPython.

I'm not sure why you see making a PR to CPython to fix this issue as being in contradiction with my proposal at https://github.com/python/cpython/issues/91533. I think taking 10 minutes to propose a PR adding a link to the CPython docs would be greatly beneficial to that effort. This is especially the case, given that the consensus in https://github.com/python/cpython/issues/91533 (which I have come to agree with, even though it was not my initial position), is that the best home for the series of HOWTOs I propose in that issue would be https://typing.readthedocs.io/en/latest/# rather than the CPython docs.

**Reply by adam-grant-hendry** [reply]
That's fine by me!

I was making an assumption (apologies). I didn't want to propose something to have it rejected by Guido and others due to there already being an ongoing effort. I don't think I realized the consensus was to add to the [typing readthedocs](https://typing.readthedocs.io/en/latest/#).

I can submit a proposal, but I can't commit to implementing it.

**Reply by adam-grant-hendry** [reply]
Done! (See [#96917](https://github.com/python/cpython/issues/96917))
